nextcord/http.py

```diff
@@ -1856,6 +1856,21 @@ def ban(
             retry_request=retry_request,
         )
 
+    def bulk_ban(
+        self,
+        user_ids: List[Snowflake],
+        guild_id: Snowflake,
+        delete_message_seconds: int = 0,
+        reason: Optional[str] = None,
+        *,
+        auth: Optional[str] = MISSING,
+        retry_request: bool = True,
+    ) -> Response[guild.BulkBan]:
+        r = Route("POST", "/guilds/{guild_id}/bulk-ban", guild_id=guild_id)
+        params = {"users_ids": user_ids, "delete_message_seconds": delete_message_seconds}
+
+        return self.request(r, params=params, reason=reason, auth=auth, retry_request=retry_request)
```

Comment:
```suggestion
        data = {"user_ids": user_ids, "delete_message_seconds": delete_message_seconds}

        return self.request(r, json=data, reason=reason, auth=auth, retry_request=retry_request)
```
These parameters must be passed as json and the parameter name is `user_ids`, not `users_ids`.

```
nextcord.ext.commands.errors.CommandInvokeError: Command raised an exception: HTTPException: 400 Bad Request (error code: 50035): Invalid Form Body
In _errors: [{'code': 'CONTENT_TYPE_INVALID', 'message': 'Expected "Content-Type" header to be one of {\'application/json\'}.'}]
```
```
nextcord.errors.HTTPException: 400 Bad Request (error code: 50035): Invalid Form Body
In user_ids: This field is required
```